### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/temporian/actions/runs/6367779359/attempts/1#summary-17286629367

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/temporian/actions/runs/6367779419/attempts/1#summary-17286629746

### CODE REVIEW

File: temporian/beam/io/test/tensorflow_test.py [hunk 1]
```diff
@@ -63,10 +63,10 @@ def test_to_and_from_tensorflow_record(self):
             )
             p.run()
 
-        loaded_evtset = in_process_from_tensorflow_record(
+        loaded_evset = in_process_from_tensorflow_record(
```
Comment (jsoref) [inline temporian/beam/io/test/tensorflow_test.py:66]: both `evset` and `evtset` were used, but the former is used way more often and there doesn't seem to be any explanation as to why the extra `t` belongs...

File: temporian/core/event_set_ops.py [hunk 2]
```diff
@@ -1149,7 +1149,7 @@ def experimental_fast_fourier_transform(
 
         The window length is defined in number of events, instead of
         timestamp duration like most other operators. The 'num_events' argument
-        needs to be specified by warg i.e. fast_fourier_transform(num_events=5)
+        needs to be specified by kwarg i.e. fast_fourier_transform(num_events=5)
```
Comment (jsoref) [inline temporian/core/event_set_ops.py:1152]: i'm pretty sure this is the pythonic thing...

File: tools/build.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Unfortunately, building non-pure packages through Poetry is still somewhat
 unstable. Notably, the build uses a custom build script located under
 `config/build.py`. The definition of these custom build scripts is being
 discussed in this
-[Github issue](https://github.com/python-poetry/poetry/issues/2740) on the
+[GitHub issue](https://github.com/python-poetry/poetry/issues/2740) on the
```
Comment (jsoref) [inline tools/build.md:16]: brand

File: tools/build.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ where `<version>` is one of `38`, `39`, `310`, `311`.
 This will place the manylinux package in the `dist/` directory under
 Temporian's root.
 
-## MacOS builds
+## macOS builds
```
Comment (jsoref) [inline tools/build.md:48]: brand

**Comment by ianspektor** [comment]
Hey @jsoref! Sorry for the huge delay, there were some security implications of having an external PR merged that we hadn't been able to sort out yet. We can merge this if you resolve the conflicts in the notebooks. Thanks for your contribution!

**Comment by jsoref** [comment]
@ianspektor: no worries; done

**Review by ianspektor — APPROVED** [review]

**Comment by ianspektor** [comment]
Thank you!

**Pull request merged**